tests/test-unsat-core-reducer.cpp

```diff
@@ -60,6 +60,23 @@ TEST_P(UnsatCoreReducerTests, UnsatCoreReducer)
   EXPECT_TRUE(rem[0] != red[0]);
 }
 
+TEST_P(UnsatCoreReducerTests, UnsatCoreReducerLinear)
+{
+  UnsatCoreReducer uscr(r);
+
+  Term a = s->make_symbol("a", boolsort);
+  Term b = s->make_symbol("b", boolsort);
+  Term formula = s->make_term(And, a, b);
+  TermVec assump({ s->make_term(Not, a), s->make_term(Not, b) });
+  TermVec red, rem;
+
+  uscr.linear_reduce_assump_unsatcore(formula, assump, red, &rem);
+  EXPECT_TRUE(red.size() == 1);
```

Comment:
This isn't too important, but it's more idiomatic to use:
`EXPECT_EQ(<arg1>, <arg2>)` and
`EXPECT_NE(<arg1>, <arg2>)`